xkit.js

```diff
@@ -3626,7 +3653,21 @@ function show_error_script(message) {
 
 function show_error_reset(message) {
 	// Shortcut to call when there is a javascript error.
-	XKit.window.show("XKit ran into an error.", "<b>Generated Error message:</b><br/><p>" + message + "</p>It is recommended that you reset XKit. Alternatively, you can write down the error message above and contact New XKit Support to see how you can fix this or reload the page to try again.", "error", "<div id=\"xkit-close-message\" class=\"xkit-button default\">OK</div><a href=\"http://www.tumblr.com/xkit_reset\" class=\"xkit-button\">Reset XKit</a><a href=\"https://new-xkit-support.tumblr.com\" class=\"xkit-button\">New XKit Support</a>");
+	XKit.window.show(
+		"XKit ran into an error.",
+
+		"<b>Generated Error message:</b><br/><p>" + message + "</p>" +
+		"It is recommended that you force-update XKit. " +
+		"Alternatively, you can write down the error message above and contact New XKit Support to see how you can fix this or reload the page to try again.",
+
+		"error",
+
+		'<div id="xkit-close-message" class="xkit-button">OK</div>' +
+		'<div id="xkit-force-update" class="xkit-button default">Force Update</div>' +
+		'<a href="http://www.tumblr.com/xkit_reset" class="xkit-button">Reset XKit</a>' +
```

Comment:
let's just remove this button—if we need someone to do it as part of support, we can just link them to it.

(and if someone wants to do it for kicks, there are easier ways then this error message)